lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -25,6 +25,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>12</LangVersion>
+    <AnalysisLevel>latest-Recommended</AnalysisLevel>
```

Comment:
Perhaps fix this to `net8.0-Recommended`. Otherwise compilation can break if one compiles using a newer SDK with a new analyzer